js/renovation/button.tsx

```diff
@@ -99,9 +106,10 @@ export const viewFunction = (viewModel: Button) => (
 export default class Button extends Widget {
     @Prop() classNames?: string[];
     @Prop() contentRender?: any;
-    @Prop() icon?: string;
+    @Prop() icon?: string = '';
```

Comment:
Why we defined it?

BTW, if we have a default option here, we should have the same default prop in the `**.j.jsx` file.